includes/admin/class-wp-job-manager-cpt.php

```diff
@@ -329,6 +331,7 @@ public function jobs_by_category() {
 			),
 		);
 
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No changes or data exposed based on input.
 		$selected_category = isset( $_GET['job_listing_category'] ) ? sanitize_text_field( wp_unslash( $_GET['job_listing_category'] ) ) : '';
```

Comment:
This along with other similar checks could be potentially refactored to a single method.

something like `<somename>::get_sanitized_var( $var, $default='' );`

Reply:
WPCS sniffs for those (or similar) functions. We could create a custom _global function_ (not class method) and set the `customUnslashingSanitizingFunctions` in PHPCS, but I've been trying to keep new global functions to a minimum.